src/components/AnnotationLineConnector/AnnotationLineConnector.scss

```
@@ -0,0 +1,16 @@
+#line-connector-root {
+    position: relative;
+    z-index: 999;
+  }
+  
+.lineHorizontal {
+background-color: rgba(30,120,235, 0.5);;
+position: fixed;
+height: 2px;
+}
+
+.lineVertical {
+background-color: rgba(30,120,235, 0.5);;
+position: fixed;
+width: 2px;
+}
```

Format is a bit off here.
```suggestion
#line-connector-root {
  position: relative;
  z-index: 999;
}

.lineHorizontal {
  background-color: rgba(30, 120, 235, 0.5);
  position: fixed;
  height: 2px;
}

.lineVertical {
  background-color: rgba(30, 120, 235, 0.5);
  position: fixed;
  width: 2px;
}

```